portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,22 +38,26 @@
   //constants for Datastore entity and property names
   private static final String ENTITY_COMMENT = "Comment";
   private static final String PROPERTY_CONTENT = "content";
+  private static final String PROPERTY_TIMESTAMP = "timestamp";
+  private static final String PROPERTY_COMMENT_NAME = "name";
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Integer maxComments = getMaxComments(request);
     if (maxComments == null) {
-        maxComments = 0;
+      maxComments = 0;
     }
```

Comment:
nit: maybe have getMaxComments return 0 instead of null, to avoid this?

Reply:
got it!